dask/dataframe/io/tests/test_orc.py

```diff
@@ -0,0 +1,53 @@
+import os
+import pytest
+import shutil
+
+from dask.dataframe import read_orc
+from dask.dataframe.utils import assert_eq
+import dask.dataframe as dd
+
+pytest.importorskip('pyarrow.orc')
+url = ('https://github.com/apache/orc/blob/master/examples/'
+       'TestOrcFile.testDate1900.orc?raw=true')
+
+
+@pytest.mark.network
+def test_orc_with_backend():
+    d = read_orc(url)
+    assert d.columns.tolist() == ['time', 'date']
+    assert len(d) == 70000
+
+
+@pytest.fixture()
+def orc_files(tmpdir):
+    requests = pytest.importorskip('requests')
+    data = requests.get(url).content
```

Comment:
This makes this test use the network too, was wondering to include the file itself

Reply:
I think I'd rather keep the test file out of our git repo, so this seems better to me. At some point I hope to wrap the writer as well which would make testing easier for us.